components/common-go/experiments/configcat.go

```diff
@@ -6,16 +6,18 @@ package experiments
 
 import (
 	"context"
+	"time"
+
 	configcat "github.com/configcat/go-sdk/v7"
 	"github.com/gitpod-io/gitpod/common-go/log"
 	"github.com/sirupsen/logrus"
-	"time"
 )
 
 const (
 	projectIDAttribute = "project_id"
 	teamIDAttribute    = "team_id"
 	teamNameAttribute  = "team_name"
+	clientIdAttribute  = "client_id"
```

Comment:
Does it make sense to use the `vscode-client-id`? To differentiate other client IDs possibly?